Application/Frontend/components/Navbar/Navbar.tsx

```diff
@@ -148,7 +152,7 @@ export function Navbar() {
                         Start chatting with your friends on the browser TODAY!
                       </Text>
                     </div>
```

Comment:
The `Text` element is actually mandatory because it comes from the `Mantine` components library, meaning that it is compatible with the theme toggle. Removing it would mean the text no longer changes from black to white upon switching from light mode to dark mode and vice versa.